test/src/test.fixture.js

```diff
@@ -61,6 +64,7 @@ class ExchangeDeviceModel extends DeviceModel {
 const nano_environments: ExchangeDeviceModel[] = [
     { name: 'nanos', letter: 'S', path: NANOS_ELF_PATH, libs: get_applications('nanos'), sdk: '2.1'},
     { name: 'nanox', letter: 'X', path: NANOX_ELF_PATH, libs: get_applications('nanox'), sdk: '2.0.2'},
+    { name: 'nanosp', letter: "SP", path: NANOSP_ELF_PATH, libs: get_applications('nanosp')},
```

Comment:
If the `sdk` field is never used, it should be cleaned in the other environment, else it should be explicited (currently 1.0.2)

Reply:
The thing is that I don't actually know which version of Speculos Zemu currently uses.
I tried several options that do NOT work : "1.0.2", "1.0.0", "1"
Everytime Speculos returns an error "invalid SDK version"

Reply:
Ok sdk version "1.0" did work so I'll use that